### PR TITLE
Propose RFC 015 - Small Group Comradro

### PR DESCRIPTION
**Pull request by zuren**
Propose and join fortnightly small group retrospectives/workshops/discussions/other on topics of your choosing.

### CODE REVIEW

File: rfc-015-small-group-comradro.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+There is no organisation currently about how Small Group Comradros should be organised, by who, how to join them, where to keep track of them, and when they should be run.
+
+## Proposal
+
+Anyone interested in proposing a Small Group Comradro **SHOULD** post their idea as a ticket in the Upcoming column of [the Trello Board][sgc-trello] by lunchtime Friday on a Comradro week, assigning themselves to it.
+
+Anyone interested in joining a Small Group Comradro **SHOULD** assign themselves to **ONE** proposal they're interested in by lunchtime Friday on a Comradro week.
+
+Proposals **SHOULD** be mentioned at the start of Learn Tech by each team, as people get used to the new format.
```
Comment (craigjbass) [inline rfc-015-small-group-comradro.md]: Is this inclusive to nontech part of Made Tech ?

Comment (zuren) [reply rfc-015-small-group-comradro.md]: No. I guess really people should know to check the Slack, expected this to be a short term reminder more than anything. Slack post as well?

**Comment by kjdchapman** [comment]
> Anyone interested in proposing a Small Group Comradro **SHOULD** post their idea as a ticket
> Proposals **SHOULD** be mentioned at the start of Learn Tech

I would bump up these to **MUST**, as I think they're essential for getting the thing running.

(great work running with this btw!)

File: rfc-015-small-group-comradro.md (hunk 1 above)
Comment (kjdchapman) [reply rfc-015-small-group-comradro.md]: Could be generic in this RFC to get things rolling?

> Proposals **MUST** be made public (e.g. slack post, mentioning at QT/learn tech)

**Pull request merged**